packages/eas-cli/src/commands/submit.ts

```diff
@@ -91,12 +92,20 @@ export default class Submit extends EasCommand {
 
   async runAsync(): Promise<void> {
     const { flags: rawFlags } = await this.parse(Submit);
-    const flags = await this.sanitizeFlagsAsync(rawFlags);
+
+    const flags = await Submit.sanitizeFlags(rawFlags);
```

Comment:
I think this is not an async function:
```suggestion
    const flags = Submit.sanitizeFlags(rawFlags);
```